src/utils.ts

```diff
@@ -26,14 +27,24 @@ export async function buildCSSInjectionCode({
     injectCodeFunction,
     useStrictCSP,
     buildOptions,
+    relativeCSSInjection,
 }: BuildCSSInjectionConfiguration): Promise<OutputChunk | null> {
     let { minify, target } = buildOptions;
 
+    // Ensures styleId is unique when relativeCSSInjection is enabled
+    const generatedStyleId = styleId && relativeCSSInjection ? `${styleId ? `${styleId}-` : ''}${uuidv4()}` : styleId ;
```

Comment:
Check the length of styleId maybe can be an empty string? I can't check this right now, maybe later